std/os/index.zig

```diff
@@ -1135,15 +1134,26 @@ pub const Dir = struct {
             SymLink,
             File,
             UnixDomainSocket,
+            Wht, // TODO wtf is this
```

Comment:
ah, thanks for informing me. Yes, let's rename this to `Whiteout` and delete the comment.